.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,18 @@
+name: 💡 Feature Request
+about: An idea or suggestion for a new feature or enhancement to the Checkout Sheet Kit
+labels:
+  - 'Feature-Request'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Hi there!
+
+        Please tell us about the new feature or enhancement you'd like to see implemented in the Checkout Sheet Kit.
+
+  - type: textarea
+    attributes:
+      label: Description
```

Comment:
Could there be any benefit in splitting this into two `description` and `rationale` sections?

Reply:
Good idea. Might also be nice to see some suggested code examples